File: 3477-fruits-into-baskets-ii/3477-fruits-into-baskets-ii.py
class Solution:
    def numOfUnplacedFruits(self, fruits: List[int], baskets: List[int]) -> int:
        n, match = len(baskets), 0
        for fruit in fruits:
            for i in range(n):
                if fruit <= baskets[i]:
                    baskets[i] = 0
                    match += 1
                    break

        return len(fruits) - match

File: 3477-fruits-into-baskets-ii/3477-fruits-into-baskets-ii.java
class Solution {
    public int numOfUnplacedFruits(int[] fruits, int[] baskets) {
        int n = baskets.length, match = 0;
        for(var fruit : fruits)
            for(var i=0; i<n; i++)
                if(fruit <= baskets[i]){
                    baskets[i] = 0;
                    match++;
                    break;
                }

        return fruits.length - match;
    }
}